volmdlr/step.py

```diff
@@ -16,6 +16,8 @@
 import plot_data.graph
 
 import webbrowser
+from jinja2 import Environment, PackageLoader, select_autoescape
```

Comment:
Remove jinja2 import, it makes difficult to install volmdlr

Reply:
Fixed in https://github.com/Dessia-tech/volmdlr/pull/190/commits/3c49b6ae958e2591dd326b81e3fd694d4084c141